Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Beckn/OnSelect.hs

```diff
@@ -110,7 +110,8 @@ onSelect OnSelectValidatedReq {..} = do
       let lowestFareQuote = selectLowestFareQuote quotes
       case lowestFareQuote of
         Just autoAssignQuote -> do
-          dConfirmRes <- SConfirm.confirm person.id autoAssignQuote.id
+          let dConfirmReq = SConfirm.DConfirmReq {personId = person.id, quoteId = autoAssignQuote.id, paymentMethodId = Nothing} -- FIXME mbPaymentMethodId always Nothing
```

Comment:
In this case, can we capture the paymentMethodId from ui/select API and save it and then use in case of auto assign.

Reply:
Added `paymentMethodId` to ui/select API , stored to `searchRequest` table.